.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+
+jobs:
+  build:
```

Comment:
This release has become unconditional, if I understand correctly - personally I would prefer to ensure the tests have run first, but 🤷